### PR TITLE
Update polkadot.js

### PR DESCRIPTION
**Pull request by yrong**
Or smoke tests on Westend will fail as follows:

```
➜  operations git:(main) ✗ pnpm transferToEthereum
...

2024-11-04 05:43:24             VEC: Unable to decode on index 0 createType(ExtrinsicUnknown):: Unsupported unsigned extrinsic version 5
2024-11-04 05:43:24        RPC-CORE: getBlock(hash?: BlockHash): SignedBlock:: createType(SignedBlock):: Struct: failed on block: {"header":"Header","extrinsics":"Vec<Extrinsic>"}:: Struct: failed on extrinsics: Vec<Extrinsic>:: createType(ExtrinsicUnknown):: Unsupported unsigned extrinsic version 5
```

### CODE REVIEW

**Review by claravanstaden — APPROVED** [review]
Nice! Maybe it will even resolve some of the intermittent Polkadot extrinsic failures I've being seeing during e2e testnet startup.

**Review by alistair-singh — COMMENTED** [review]
+1

File: web/packages/api/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@snowbridge/api",
-  "version": "0.1.23",
+  "version": "0.1.24",
```
[the remark this answers is not in the document]
Comment (yrong) [reply web/packages/api/package.json:3]: Just curious is that required? We did not upgrade any dependency/code in above two packages.

[the remark this answers is not in the document]
Comment (alistair-singh) [reply web/packages/api/package.json:3]: The release workflow fails if you try to publish the same version of the same packages. So I version all 3. The release scripts are not smart enough to detect this and skip publishing.

[the remark this answers is not in the document]
Comment (yrong) [reply web/packages/api/package.json:3]: https://github.com/Snowfork/snowbridge/pull/1328/commits/ac248027fe5bc658b799beb317d673a6cb37957d

**Pull request merged**